### PR TITLE
Update minimist to 1.2.2

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
https://snyk.io/vuln/SNYK-JS-MINIMIST-559764
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7598

## How Has This Been Tested?
- :robot: 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: build/yarn.lock [hunk 1]
```diff
@@ -1598,6 +1598,11 @@ minimist@0.0.8:
...
   integrity sha1-hX/Kv8M5fSYluCKCYuhqp6ARsF0=
 
+minimist@1.2.2:
+  version "1.2.2"
+  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.2.tgz#b00a00230a1108c48c169e69a291aafda3aacd63"
+  integrity sha512-rIqbOrKb8GJmx/5bc2M0QchhUouMXSpd1RTclXsB41JdL+VtnojfaJR+h7F9k18/4kHUsBFgk80Uk+q569vjPA==
+
 minimist@^1.1.3:
   version "1.2.0"
   resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.0.tgz#a35008b20f41383eec1fb914f4cd5df79a264284"
```
Comment (phil-davis) [inline build/yarn.lock]: what are these lines about, that mention 1.2.0 - will something still use 1.2.0, or?

Comment (DeepDiver1975) [reply build/yarn.lock]: as far as I understand the lock file all depending versions of each dependency is listed.
but only 1.2.2 is installed - I did verify that

**Comment by DeepDiver1975** [comment]
ci execution has some hickups - unrelated -> merge if there are no objections

**Comment by phil-davis** [comment]
> ci execution has some hickups - unrelated -> merge if there are no objections

No objections from me. The fails are stupid slow pulls of database docker images, and happened in API acceptance test setup. This PR changes dependencies that are completely unrelated to the backend PHP and APIs.

**Pull request merged**